models/intermediate/int_amazon_selling_partner__item.sql

```diff
@@ -0,0 +1,179 @@
+with item_summary as (
+    select *
+    from {{ ref('stg_amazon_selling_partner__item_summary') }}
+),
+
+item_product_type as (
+
+    select *
+    from {{ ref('stg_amazon_selling_partner__item_product_type') }} 
+),
+
+item_image as (
+
+    select *
+    from {{ ref('stg_amazon_selling_partner__item_image') }}
+),
+
+item_images as (
+
+    select 
+        source_relation,
+        asin,
+        marketplace_id,
+        count(*) as count_images,
+        sum(case when variant = 'SWATCH' then 1 else 0 end) as count_swatch_images
```

Comment:
Please use the lower function and then `swatch` here to ensure future maintainability.

Reply:
Please apply the same to other string case statements.

Reply:
using upper to maintain consistency with the amazon source API